### DISCUSSION TITLE
Additional Error logs for Level Batching

### DISCUSSION THREAD

**Discussion by Spebby** [SmartlyDressedGames/Unturned-3.x-Community#3719] [Ideas]
While updating Easter Island to better support batching, I noticed that a lot of materials weren't being included due to having emissions set on, but no map and with a black colour as the base. Visually there'd be no effect, but they couldn't be batched. Tracking these specific objects down was quite tedious, and it would be helpful if the game warned about them.

In addition, on Objects where the materials were seemingly fine, I noticed that even if the object does not repeat it texture at all, as long as the texture in unity has it enabled, the game will not be able to batch it. An error log for this (ideally a toggleable one since the game wouldn't be able to differentiate between objects that use repeating textures vs those that just have it enabled, and error logs would get cluttered)

A error log for mipmaps being enabled on textures under 16x16 would also be appreciated.

**Comment by SDGNelson** [comment]
These kind of warnings are logged with the **-LogLevelBatchingTextureAtlasExclusions** launch option.
